### PR TITLE
Merge thinkhazard_common and thinkhazard_processing in thinkhazard repository.

### PR DESCRIPTION
**Pull request by arnaud-morvan**
* Remove dependency to thinkhazard_common.
* Add all tasks from thinkhazard_processing.

Note that processes now use database url configuration from development.ini file or optional local.ini file. `thinkhazard_processing.yaml` is there only for processing specific configuration.

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
@pgiraud @fvanderbiest : Could you review it.

File: thinkhazard/scripts/harvest.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+              been fetched')
+    parser.add_argument(
+        '--dry-run', dest='dry_run',
+        action='store_const', const=True, default=False,
+        help='Perform a trial run that does not commit changes')
+    args = parser.parse_args(argv[1:])
+
+    engine = engine_from_config(settings, 'sqlalchemy.')
+    import pydevd
+    pydevd.settrace()
```
Comment (pgiraud) [inline thinkhazard/scripts/harvest.py]: The 2 lines above are probably meant to be removed (debug?).

**Comment by pgiraud** [comment]
How is it possible to change the `data_path`?

**Comment by pgiraud** [comment]
Just 2 minor comments, otherwise looks good to me.
Please merge when addressed.

**Comment by arnaud-morvan** [comment]
data_path configuration key is documented in readme file:
https://github.com/GFDRR/thinkhazard/blob/merge_repositories/README.rst#data_path

Note that this has no effect for administrative divisions where import is made at make time.

**Comment by pgiraud** [comment]
This means that you need to modify the `processing_settings.yaml` file.

**Comment by pgiraud** [comment]
Looks good. Please merge.

**Pull request merged**